apps/dashboard/src/components/subscribers/preferences/subscriber-preferences.tsx

```diff
@@ -0,0 +1,135 @@
+import { showSuccessToast } from '@/components/primitives/sonner-helpers';
+import { Tooltip, TooltipContent, TooltipTrigger } from '@/components/primitives/tooltip';
+import { SidebarContent } from '@/components/side-navigation/sidebar';
+import { SubscriberPreferencesSkeleton } from '@/components/subscribers/preferences/subscriber-preferences-skeleton';
+import { SubscriberChannelToggle } from '@/components/subscribers/preferences/subscriber-preferences-toggles';
+import { SubscriberWorkflowPreferences } from '@/components/subscribers/preferences/subscriber-workflow-preferences';
+import useFetchSubscriberPreferences from '@/hooks/use-fetch-subscriber-preferences';
+import { usePatchSubscriberPreferences } from '@/hooks/use-patch-subscriber-preferences';
+import { GetSubscriberPreferencesDto, PatchPreferenceChannelsDto } from '@novu/api/models/components';
+import { ChannelTypeEnum } from '@novu/shared';
+import { motion } from 'motion/react';
+import { useEffect, useMemo, useState } from 'react';
+import { RiQuestionLine } from 'react-icons/ri';
+
+export default function SubscriberPreferences({ subscriberId }: { subscriberId: string }) {
```

Comment:
Prefer named exports 

```suggestion
export function SubscriberPreferences({ subscriberId }: { subscriberId: string }) {
```